### PR TITLE
[metasrv] refactor: add KVApiTestSuite::test_single_node() to run all single-node tests

### PR DESCRIPTION
**Pull request by drmingdrmer**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Changelog

- Improvement

## Related Issues

### CODE REVIEW

File: metasrv/tests/it/main.rs [hunk 1]
```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// Because the compiler complains about recursion limit for a trait requirement check...
```
Comment (ariesdevil) [inline metasrv/tests/it/main.rs:15]: There should be more comments or a reference link here to explain this case.

**Comment by ariesdevil** [comment]
Rest look good to me

File: metasrv/tests/it/main.rs (hunk 1 above)
Comment (drmingdrmer) [reply metasrv/tests/it/main.rs:15]: Yes sir~

**Review by ariesdevil — APPROVED** [review]
LGTM~

**Comment by ariesdevil** [comment]
The failed tests were not related to this PR:)

**Pull request merged**